sdk/core/azure-core/CHANGELOG.md

```diff
@@ -16,6 +16,13 @@
 ### Other Changes
 
 - [[#3581]](https://github.com/Azure/azure-sdk-for-cpp/issues/3581) Update log level in retry policy from warning to informational.
+- Support OpenSSL 3.0.
```

Comment:
```suggestion
- Support OpenSSL 3.0. (A community contribution, courtesy of _[jepio](https://github.com/jepio)_)
```

Reply:
What's the implications for the SDK customer with this addition? We are stating we are supporting a newer version in the CL, but it would be useful to provide info to the reader around: how does the customer get impacted here? Why would they care?